### ISSUE TITLE
Древний баг на который никто не жаловался

### ISSUE THREAD

**Issue by Motzord** [TauCetiStation/TauCetiClassic#9949]
#### Подробное описание проблемы
В лобби слышны звуки спавна на велосити (звук копошения рюкзака/педального обувания эквипа)
#### Что должно было произойти
Не слышно
#### Что произошло на самом деле
Слышно
#### Как повторить
Сидите в лобби
Слышите как кто-то заходит поиграть

**Comment by L4rever** [comment]
Это баг?

**Comment by BunkyFly** [comment]
> Это баг?

Да, потому что в лобби все персонажи спавнятся и одеваются, а потом их телепортирует на Велосити
